### ISSUE TITLE
Markdown files with attachments not imported properly

### ISSUE THREAD

**Issue by CR0CKER** [laurent22/joplin#5446]
When exporting notes with attachments from Bear Notes as markdown, attached files are saved in a subfolder of the same name as the MD file, and linked locally from the Markdown file. However, when importing these files into Joplin, none of the attached files are imported and the Joplin notes end up with dead links and no attachments. Since almost every markdown editor that supports attachments saves them like this, it seems importing these MD files should work in Joplin.

Interestingly, the attached files actually are stored inside Joplin, because they show up in the list of attachments when going to Tools > Note attachments. The files can be opened. But they don't seem to get linked properly from the respective note. 

## Environment

Joplin version: 2.3.5
Platform: MacOS Big Sur &  Linux
OS specifics: Ubuntu
Bear Notes: 1.8.5

## Steps to reproduce

1. Export note with attachment from Bear as markdown
2. Attachments are linked locally like this:
`
![](Be%20my%20bikey/image_1.jpeg)
`
3. Import note into Joplin
4. Attachment ends up in Joplin note like this but without actually having the file available under the given path:
`
![](Be%20my%20bikey/image_1.jpeg)
`
5. The attachment can also be found in the "Note attachments" list in Joplin. But it is not shown inside the note. 

## Describe what you expected to happen

I would expect all attached files, no matter which format, to be visible when looking at the imported note.

## Logfile

`DevTools failed to load SourceMap: Could not load content for file:///Applications/Joplin.app/Contents/Resources/app.asar/node_modules/@joplin/lib/services/interop/InteropService_Importer_Md.js.map: 
/Applications/Joplin.app/Contents/Resources/app.asar/node_modules/@joplin/lib/Logger.js:188 21:50:23: Scheduling sync operation... 30000
/Applications/Joplin.app/Contents/Resources/app.asar/gui/MenuBar.js:188 Import result:  {warnings: Array(0)}
/Applications/Joplin.app/Contents/Resources/app.asar/node_modules/@joplin/lib/Logger.js:188 21:50:23: CommandService::execute: hideModalMessage
/Applications/Joplin.app/Contents/Resources/app.asar/node_modules/@joplin/lib/Logger.js:188 21:50:33: SearchEngine: Updating FTS table...
/Applications/Joplin.app/Contents/Resources/app.asar/node_modules/@joplin/lib/Logger.js:188 21:50:33: Saving settings...
/Applications/Joplin.app/Contents/Resources/app.asar/node_modules/@joplin/lib/Logger.js:188 21:50:33: Settings have been saved.
/Applications/Joplin.app/Contents/Resources/app.asar/node_modules/@joplin/lib/Logger.js:188 21:50:33: SearchEngine: Updated FTS table in 67ms. Inserted: 1. Deleted: 0`

**Comment by CR0CKER** [comment]
Here you go!
[Example.zip](https://github.com/laurent22/joplin/files/7136823/Example.zip)

**Comment by JackGruber** [comment]
Under windows the file is imported and displayed without any problems.
![grafik](https://user-images.githubusercontent.com/24863925/132702405-bbd45f27-942e-487b-8c16-ddb6e6ca1d0f.png)

Currently, I can't try it under Linux.

**Comment by CR0CKER** [comment]
Thanks, Jack -- that's good to know and also odd. I'll also try it on Linux. Definitely not working on MacOS.

**Comment by CalebJohn** [comment]
There were some updates to the markdown importer in v2.4.2 (currently in pre-release). @CR0CKER do you mind testing with the [latest pre-release](https://github.com/laurent22/joplin/releases/tag/v2.4.5)? If not I'll be able to test later, but I suspect that moving to the pre-release will solve your problem.

**Comment by JackGruber** [comment]
Just as info, under Windows I tested it with Joplin 2.4.3

**Comment by CR0CKER** [comment]
Hey, the pre-release works! Problem solved :) Thanks for your help, all.

**Comment by jeff47** [comment]
Hmm, I am still seeing a similar error in Joplin 2.7.15.  The attachment appears when I look in Joplin (Note Attachments), but it's a broken link in the note.

**Comment by kshunterco** [comment]
> Hmm, I am still seeing a similar error in Joplin 2.7.15. The attachment appears when I look in Joplin (Note Attachments), but it's a broken link in the note.

Jumping in quite late but echoing @jeff47 's comment - I have markdown with a _resource directory (generated by evernote2obsidian exporting a evernote-backup SQLite backup database) and when I import it to Joplin... I get nada. Nothing shows up in Tools->Note Attachments. Tried putting the _resource directory in different places w/respect to the directory with md... no change in behavior.

_Edit:_ well crap I just tried the example import and it worked like a damn charm so it must be on my end!!! Argh
